.changeset/silver-pigs-care.md

```diff
@@ -0,0 +1,12 @@
+---
+'@ag.ds-next/react': minor
+---
+
+box: Add `focusAll` prop to enable a focus ring when the element is programmatically focused.
+
+drawer: Add `elementToFocusOnClose` to allow custom element to be focused when the drawer closes.
+
+page-alert: Add `isHidden` prop to hide alert with CSS so that its `ref` can be passed to the drawer's `elementToFocusOnClose`. Also enable the newly added `focusAll` prop from `Box` to improve accessibility for programmatically focused alerts.
+
+page-alert: Add `isHidden` prop to hide alert with CSS so that its `ref` can be passed to the drawer's `elementToFocusOnClose`. Also enable the newly added `focusAll` prop from `Box` to improve accessibility for programmatically focused alerts.
```

Comment:
```suggestion
section-alert: Add `isHidden` prop to hide alert with CSS so that its `ref` can be passed to the drawer's `elementToFocusOnClose`. Also enable the newly added `focusAll` prop from `Box` to improve accessibility for programmatically focused alerts.
```

Reply:
- [x] Update changeset